### DISCUSSION TITLE
How to UNCOMPRESS

### DISCUSSION THREAD

**Discussion by NakhyunKim1987** [sqlalchemy/sqlalchemy#8046] [Usage Questions]
Hello.

I want to know how to release the compressed text.

According to the results, the compressed string is all empty.

I'd like to know the solution.

### models.py

` class MapPoint(Base): 
__tablename__ = "mapPoint"
    seq = Column(Integer, primary_key=True)
    drivingLogSeq = Column(Integer)
    locations = Column(String)`
 

###  main.py
queryStr = f"  drivingLogSeq, UNCOMPRESS(locations) AS locations FROM mapPoint"
  
    drivingLogs = (
        db.query(text(queryStr))
        .limit(2)
        .all() 
    )
   
result:
[(250136,), (250138,)]

**Comment by zzzeek** [comment]
if UNCOMPRESS is a SQL function, take a look at https://docs.sqlalchemy.org/en/14/tutorial/data_select.html#working-with-sql-functions for a tutorial on how SQL functions are used.
